### PR TITLE
Add Erlang example

### PR DESCRIPTION
**Pull request by ElyKar**

### CODE REVIEW

File: src/erlang/duck.erl [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+main() ->
+    TheWolf = wolf,
+    TheDuck = TheWolf,
+    ADuck = mallard,
+
+    twoDucksAlone(ADuck, TheDuck),
+
+    io:fwrite(eat(TheWolf, ADuck)).
+
+eat(wolf, _) -> " 😈 ~n".
```
Comment (arnobl) [inline src/erlang/duck.erl]: Unicode chars do not print correctly like that.
Please, surround the string literal with unicode:characters_to_binary( ):
unicode:characters_to_binary(" 😈 ~n")

File: src/erlang/duck.erl [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+quack(wolf) -> "QUACK QUACK WHOO~n";
+quack(mallard) -> "quack quack~n".
+
+dance(wolf) -> " ¯\\_()_/¯ ~n";
```
Comment (arnobl) [inline src/erlang/duck.erl]: Idem here:
unicode:characters_to_binary(" ¯\\_()_/¯ ~n")

File: src/erlang/duck.erl [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+dance(mallard) -> " _/¯ ~n".
```
Comment (arnobl) [inline src/erlang/duck.erl]: Idem here: unicode:characters_to_binary(" _/¯ ~n")

**Comment by arnobl** [comment]
Thx for the contrib! Minor changes to apply before a merge.

**Comment by ElyKar** [comment]
I'd be happy to apply them !

**Comment by arnobl** [comment]
Can you see the changes to apply?

**Comment by ElyKar** [comment]
I assume the solution would be to make TheWolf, ADuck and TheDuck records holding their behavior. TheWolf would be a `wolf` record, while ADuck and TheDuck would be `duck` records.

**Comment by arnobl** [comment]
I forgot to validate the review...

**Comment by ElyKar** [comment]
Fine, the changes have been committed.

**Comment by arnobl** [comment]
Thx!

**Pull request merged**